### PR TITLE
Esm

### PR DESCRIPTION
**Pull request by heapwolf**

### CODE REVIEW

**Comment by chicoxyzzy** [comment]
rebased on master

File: util.js [hunk 1]
```diff
@@ -1,6 +1,4 @@
-const { Buffer } = require('./node/buffer')
-
-const _require = typeof require !== 'undefined' && require
+import { Buffer } from 'buffer'
```
Comment (jwerle) [inline util.js:1]: Do we get `buffer` from bundler?

**Review by jwerle — APPROVED** [review]
LGTM. We get `buffer` and `events` for free? We should remove and ignore the `package-lock.json` eventually before publishing

File: bluetooth.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    })
+
+    window.addEventListener('data', e => {
+      if (e.detail.params.serviceId !== this.serviceId) return
+      this.emit(this.keys[e.detail.params.characteristicId], e.detail.data)
+    })
+  }
+
+  subscribe (key) {
+    return this.set(key)
```
Comment (jwerle) [inline bluetooth.js]: `set()` doesn't appear to be defined. Is this still just a WIP?

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**